backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/invoiceCandidatesAggrgation.feature

```diff
@@ -0,0 +1,221 @@
+@from:cucumber
+Feature: invoice generation and invoice candidates aggregation
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And set sys config boolean value true for sys config SKIP_WP_PROCESSOR_FOR_AUTOMATION
+
+  @from:cucumber
+  Scenario: sales orders with the same salesRep_IDs, one invoice with SalesRep_ID
+    Given metasfresh has date and time 2021-04-16T13:30:13+01:00[Europe/Berlin]
+    And metasfresh contains M_Products:
+      | Identifier | Name            |
+      | p_1        | salesProduct_70 |
+      | p_2        | salesProduct_80 |
+    And metasfresh contains M_PricingSystems
+      | Identifier | Name                   | Value                   | OPT.Description            | OPT.IsActive |
+      | ps_1       | pricing_system_name_70 | pricing_system_value_70 | pricing_system_description | true         |
+    And metasfresh contains M_PriceLists
+      | Identifier | M_PricingSystem_ID.Identifier | OPT.C_Country.CountryCode | C_Currency.ISO_Code | Name               | OPT.Description | SOTrx | IsTaxIncluded | PricePrecision | OPT.IsActive |
+      | pl_1       | ps_1                          | DE                        | EUR                 | price_list_name_70 | null            | true  | false         | 2              | true         |
+    And metasfresh contains M_PriceList_Versions
+      | Identifier | M_PriceList_ID.Identifier | Name              | ValidFrom  |
+      | plv_1      | pl_1                      | salesOrder-PLV_70 | 2021-04-01 |
+    And metasfresh contains M_ProductPrices
+      | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_UOM_ID.X12DE355 | C_TaxCategory_ID.InternalName |
+      | pp_1       | plv_1                             | p_1                     | 10.0     | PCE               | Normal                        |
+      | pp_2       | plv_1                             | p_2                     | 5.0      | PCE               | Normal                        |
+    And metasfresh contains C_BPartners:
+      | Identifier    | Name         | OPT.IsVendor | OPT.IsCustomer | M_PricingSystem_ID.Identifier | OPT.InvoiceRule |
+      | endcustomer_1 | Endcustomer2 | N            | Y              | ps_1                          | D               |
```

Comment:
to avoid collision with other tests pls add append a unique suffix to the business partner name, i.e. the metasfresh/metasfresh issue number